hardware/cpu/intel/xeon_gold_5218.pan

```diff
@@ -0,0 +1,10 @@
+structure template hardware/cpu/intel/xeon_gold_5218;
+
+"manufacturer" = "Intel";
+"model" = "Intel(R) Xeon(R) Gold 5120 CPU @ 2.20 GHz";
```

Comment:
The model name does not fit with the template name.

Reply:
Well spotted, thanks!